src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -110,38 +110,37 @@ library BeaconChainProofs {
 
 
     /// @notice This struct contains the merkle proofs and leaves needed to verify a partial/full withdrawal
-    struct WithdrawalProofs {
+    struct WithdrawalProof {
         bytes32 beaconStateRoot;
-        bytes latestBlockHeaderProof;
-        bytes blockHeaderProof;
+        bytes stateRootProof;
         bytes withdrawalProof;
         bytes slotProof;
         bytes executionPayloadProof;
         bytes timestampProof;
         bytes historicalSummaryBlockRootProof;
-        uint64 blockHeaderRootIndex;
+        uint64 blockRootIndex;
         uint64 historicalSummaryIndex;
         uint64 withdrawalIndex;
-        bytes32 blockHeaderRoot;
+        bytes32 blockRoot;
         bytes32 blockBodyRoot;
```

Comment:
idk man its technically the blockHeader root.  idk if we should perpetuate the misnomer

Reply:
So blockHeaderRoot is one level above the blockBodyRoot, which is why I liked the distinction.  But that being said,
blockHeader and block are basically the same:

`class BeaconBlockHeader(Container):
    slot: Slot
    proposer_index: ValidatorIndex
    parent_root: Root
    state_root: Root
    body_root: Root`
    
    `class BeaconBlock(Container):
    slot: Slot
    proposer_index: ValidatorIndex
    parent_root: Root
    state_root: Root
    body: BeaconBlockBody`
    
    
    So blockHeader is like an intermediate step to hashing block.  So I actually agree blockRoot is honestly the more clear option.

Reply:
great, then we have agreement on this. I just pushed a commit eliminating the unused 'blockBodyRoot' field as well: https://github.com/Layr-Labs/eigenlayer-contracts/pull/189/commits/e131d56f58ee455f8093a29f3e2a524ceb1e7d78